resources/lib/template.py

```diff
@@ -344,7 +344,10 @@ def getProperties( self, elem, items ):
                     continue
                     
                 # We've matched a property :)
-                properties[ name ] = property.text
+                if property.text is not None and property.text != "":
```

Comment:
"if property.text:" definitely looks more readable to me than "if property.text is not None and property.text != "":" :)
Anyways, this is just cosmetics. Push the green button. ;)